common/lib/common-definitions/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@fluidframework/common-definitions",
-	"version": "1.2.0",
+	"version": "2.0.0",
```

Comment:
Is it convention to update this to the next major version because removing these deprecated types is a breaking change?

Reply:
Yep, that was my reasoning. But note that this package is not part of the release group, that's why I did it directly in it's package json. For release groups, the versions are handled by something else.